keras_nlp/applications/bert.py

```diff
@@ -66,118 +67,118 @@ class BertModel(keras.Model):
     def __init__(
         self,
         vocab_size,
-        num_layers=12,
-        hidden_size=768,
-        dropout=0.1,
-        num_attention_heads=12,
-        inner_size=3072,
+        num_layers,
+        hidden_size,
+        num_attention_heads,
```

Comment:
we could do num_heads to agree with TransformerEncoder, but maybe the extra specificity of num_attention_heads is good in this more general context.

Reply:
I think `num_heads` seems clear in this context.